libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.kt

```diff
@@ -195,6 +195,12 @@ open class LoginActivity : FragmentActivity() {
             SalesforceSDKManager.getInstance().setViewNavigationVisibility(this)
         }
 
+        // Set the Salesforce Welcome Login hint and host for the OAuth authorize URL, if applicable.
+        viewModel.loginHint = intent.getStringExtra(EXTRA_KEY_LOGIN_HINT)
+        intent.getStringExtra(EXTRA_KEY_LOGIN_HOST)?.let {
+            viewModel.loginUrl.value = "https://$it"
+        }
+
```

Comment:
Just a thought:  It might end up being helpful down the line to add welcome to the server list.  I believe we record the server used to login when we create the account.

Reply:
Also we should add welcome.salesforce.com to the default list of servers (right now we have login.salesforce.com and test.salesforce.com). That could probably go as part of this PR.

Reply:
I meant in this file: https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/SalesforceSDK/res/xml/servers.xml